src/Metadata/Extractor/IdentifiersExtractor.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Metadata\Extractor;
+
+use ApiPlatform\Api\CompositeIdentifierParser;
+use ApiPlatform\Api\ResourceClassResolverInterface;
+use ApiPlatform\Metadata\Exception\RuntimeException;
+use ApiPlatform\Metadata\GraphQl\Operation as GraphQlOperation;
+use ApiPlatform\Metadata\HttpOperation;
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\Metadata\Property\Factory\PropertyMetadataFactoryInterface;
+use ApiPlatform\Metadata\Property\Factory\PropertyNameCollectionFactoryInterface;
+use ApiPlatform\Metadata\Resource\Factory\ResourceMetadataCollectionFactoryInterface;
+use ApiPlatform\Metadata\Util\ResourceClassInfoTrait;
+use Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+
+/**
+ * {@inheritdoc}
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+final class IdentifiersExtractor implements IdentifiersExtractorInterface
```

Comment:
class should be aliased with the if(false) hack or something, please verify that there's a functional test (in `tests/`) that uses the old class (or we'll add one)